client/js/chart/index.js

```diff
@@ -219,3 +239,44 @@ function createDefs() {
 
   return defs;
 }
+
+/**
+ * Creates a tooltip that responds to the user pointing at things on the chart.
+ *
+ * @param {SVGSVGElement} svg The SVG element for the chart.
+ * @param {Map<string, number[]>} rankingHistory The ranking history for each
+ * track.
+ * @param rankingDates
+ * @returns {HTMLDivElement} An element for the tooltip.
+ */
+function createTooltip(svg, rankingHistory, rankingDates) {
```

Comment:
how is creating a tooltip different from hit test?

Reply:
Hit test determines where the tooltip should go